File: WebApi/Interfaces/IGoodReciptFunctionsService.ts
import { ApiResponse } from '../Models/ApiResponse';
import { GoodsOpenOrdersHeaderDto, GoodsOpenOrdersLineDto } from '../Models/GoodReciptFunctionsDtos';

export interface IGoodReciptFunctionsService {
  getGoodsReceiptHeader(customerCode: string): Promise<ApiResponse<GoodsOpenOrdersHeaderDto[]>>;
  getGoodsReceiptLine(siparisNoCsv: string): Promise<ApiResponse<GoodsOpenOrdersLineDto[]>>;
}

File: Interfaces/GoodReceiptInterfaces/IGoodReciptFunctionsService.cs
using WMS_WEBAPI.DTOs;

namespace WMS_WEBAPI.Interfaces
{
    public interface IGoodReciptFunctionsService
    {
        Task<ApiResponse<List<GoodsOpenOrdersHeaderDto>>> GetGoodsReceiptHeaderAsync(string customerCode);
        Task<ApiResponse<List<GoodsOpenOrdersLineDto>>> GetGoodsReceiptLineAsync(string siparisNoCsv);
    }
}